_posts/2018-08-21-call-for-mentors-hackillinois-2019.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Call for Mentors: HackIllinois 2019!"
+date: 2018-08-21 17:00:00
+categories: press
+tags: HackIllinois2019, mentors, featured
+image: /assets/article_images/call-for-mentors-hackillinois-2019/header.jpg
+author: Justin Yang
+redirect_from: /callformentors
+---
+
+Classes begin next week here at the University of Illinois at Urbana-Champaign, and HackIllinois is ramping up its mentor outreach efforts. We're encouraging all open source contributors to fill out our [HackIllinois 2019 Mentor Interest Form](http://go.hackillinois.org/mentor).
+
+Who? Open source contributors to projects big and small
+What? A 36-hour open source hackathon that brings together 1000+ students and mentors. Work throughout the event with a small group of students on open source contributions.
+When? Friday-Sunday, February 22-24, 2019
+Where? The University of Illinois at Urbana-Champaign, Urbana, Illinois, United States
+Approximately 150 mi / 250 km from each of Chicago, Illinois; St. Louis, Missouri; and Indianapolis, Indiana
+Why? HackIllinois aims to foster education, collaboration, and mentorship within the open source community. Come help students during their first forays into open source, and meet leaders and other contributors in open source.
+How? Just fill out the Mentor Interest Form: [go.hackillinois.org/mentor](http://go.hackillinois.org/mentor)
+
+What are the commitments and responsibilities of a mentor?
+Mentors lead a group of students in addressing issues on projects that the mentors contribute to. Unlike at other collegiate hackathons, mentors don't just support students in fixing bugs here and there; mentors work together with students, guiding them through the open source workflow to make meaningful contributions.
+
+What sort of contributions have groups made in the past?
+In the past, HackIllinois has seen students make contributions to a projects in the areas of JavaScript, Android, Python, Linux, and more! Groups have opened PRs to projects such as PrairieLearn, a problem-driven learning system built at the University of Illinois; Puppeteer, Chrome's headless node API; and Rust, a systems programming language that's a favorite amongst developers. As a mentor, you can bring projects that you maintain or contribute to---anything you believe you can guide students through making contributions to.
+
+How many people will I be working with?
+Group sizes vary, but we find that the most successful groups have a ratio of 4-6 students per mentor and have 1-3 mentors that come ready with a handful features and fixes for the students to get started on.
```

Comment:
I would mention that the mentor gets to provide preference for their group size - we want them to be comfortable.